docs/SUMMARY.md

```diff
@@ -4,23 +4,25 @@
 
 * [Home](README.md)
 * [Architecture](architecture.md)
-* [Cumulus Deployment](deployment/README.md)
+* Deployment
+  * [How to Deploy Cumulus](deployment/README.md)
   * [Creating an S3 Bucket](deployment/create_bucket.md)
   * [Locating IAMs](deployment/iam_roles.md)
   * [Troubleshooting Deployment](deployment/troubleshoot_deployment.md)
-* [Cumulus Workflows](workflows/README.md)
-  * [Protocol](workflows/protocol.md)
-  * [Input & Ouptut](workflows/input_output.md)
-  * [Cumulus Task Message Flow](workflows/cumulus-task-message-flow.md)
+* [What are Cumulus Workflows?](workflows/README.md)
+  * [Workflow Protocol](workflows/protocol.md)
+  * [Workflow Tasks](tasks.md)
+  * [Workflow Input & Ouptut](workflows/input_output.md)
+  * [Workflow Task Message Flow](workflows/cumulus-task-message-flow.md)
   * [Developing Workflow Tasks](workflows/developing-workflow-tasks.md)
     * [Lambda Functions](workflows/lambda.md)
     * [Dockerization](workflows/docker.md)
   * [Workflow Configuration How-to's](workflows/workflow-configuration-how-to.md)
-* [Tasks](tasks.md)
-* [Cumulus API](https://cumulus-nasa.github.io/cumulus-api)
-* [Backup and Restore](backup_and_restore.md)
-* [EMS Reporting](ems_reporting.md)
-* [Local Docs](doc_installation.md)
+* Additional Cumulus Features
+  * [Cumulus Metadata in DynamoDB](data_in_dynamodb.md)
```

Comment:
Might it be worth still having a sub-header link under "Cumulus Metadata in DynamoDB" here that still links to the backup/restore topic?

Reply:
heh heh I didn't even see this when I wrote my comment :)

Reply:
Of course Jonathan says what I am trying to say in a much more concise way.